### PR TITLE
The family class has been added to the project and all the field   has maked private

### PR DESCRIPTION
**Pull request by thapdigov**

### CODE REVIEW

File: src/main/java/az/edu/strangers/Human.java [hunk 1]
```diff
@@ -1,53 +1,100 @@
...
     }
 
+
     public String toString() {
-        return String.format("Human{name = %s," + "surname = %s," + "year = %d," + "iq= %d," +
-                "mother = %s%s,%s",
-                name, surname, year,
-                IQ, mother.name, mother.surname, pet.toString());
+        return String.format("{ Human: name = %s, surname = %s, year = %d, IQ= %d,",
+                getName(), getSurname(), getYear(), pet.toString());
```
Comment (Shammad-Kosayev) [inline src/main/java/az/edu/strangers/Human.java]: Wrong string statement.
Documentation: " refactor the toString() method. It should output info in the following format: Human{name='Name', surname='Surname', year=1, iq=1, schedule=[[day, task], [day_2, task_2]]} "

File: src/main/java/az/edu/strangers/Pet.java [hunk 2]
```diff
@@ -3,42 +3,84 @@
...
 
+    public Pet(String species, String nickName, Integer age, Integer trickLevel, String[] habits) {
+        this.species = species;
+        this.nickName = nickName;
+        this.age = age;
+        this.trickLevel = trickLevel;
+        this.habits = habits;
+    }
 
     public void eat() {
```
Comment (Shammad-Kosayev) [inline src/main/java/az/edu/strangers/Pet.java:29]: Redundant empty line. Please delete

File: src/main/java/az/edu/strangers/Pet.java [hunk 3]
```diff
@@ -3,42 +3,84 @@
...
+    public void setHabits(String[] habits) {
+        this.habits = habits;
+    }
+
+
     public String toString() {
-        return "%s{nickname='%s', age=%d, trickLevel=%d, habits=%s}"
-                .formatted(species, nickname, age, trickLevel, Arrays.toString(habits));
+        return String.format("{ Pet: species=%s ,nickname=%s, age=%d, trickLevel=%d, habits=%s }"
+                , getSpecies(), getNickName(), getAge(), getTrickLevel(), Arrays.toString(getHabits()));
```
Comment (Shammad-Kosayev) [inline src/main/java/az/edu/strangers/Pet.java]: Wrong string statement. Old one was the correct one.

File: src/main/java/az/edu/strangers/Human.java [hunk 4]
```diff
@@ -1,53 +1,100 @@
...
+    private Family family;
 
-    public Human(){}
+    public Family getFamily() {
+        return family;
+    }
+
+    public void setFamily(Family family) {
+        this.family = family;
+    }
```
Comment (Semmed-Kosayev) [inline src/main/java/az/edu/strangers/Human.java:18]: Why getFamily and setFamily is above constructor?

File: src/main/java/az/edu/strangers/Human.java [hunk 5]
```diff
@@ -1,53 +1,100 @@
...
+
+    public Human() {
+    }
```
Comment (Shammad-Kosayev) [inline src/main/java/az/edu/strangers/Human.java]: Why getFamily and setFamily is above constructor?

**Review by Shammad-Kosayev — CHANGES_REQUESTED** [review]
Please look and apply changes

**Review by Shammad-Kosayev — APPROVED** [review]
Looks good to me

**Pull request merged**